client/java/src/test/resources/config/metrics-complex.yaml

```diff
@@ -0,0 +1,6 @@
+transport:
```

Comment:
minor: filename should be `composite` as well, right?

Reply:
Fixed.